src/api/streams/createStream.js

```diff
@@ -8,11 +8,13 @@ export default (
   description?: string = '',
```

Comment:
The last line of the commit message might be a bit premature :) (and can be omitted!)